template/Content/FeatherHttpContent.csproj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>featherhttp_template</RootNamespace>
```

Comment:
Why is this part of the template csproj?

Reply:
RootNamespace is not required and is also incorrect. I've removed the RootNamespace in my most recent commit 'cleaning up template csproj'

Please let me know if you would like additional changes here